docs/graphs/graph-queries/shortest-path.md

```diff
@@ -1,27 +1,27 @@
 ---
-sidebar_position: 2
-title: Shortest Path
+sidebar_position: 20
+title: Shortest Path Query
 ---
 
-## General idea
+A _shortest path query_ is a query that finds the shortest path between two given documents in your graph. The two documents are referred to as the `startVertex` and `targetVertex`. When you execute this query, you receive a result in the form of a set with two items for each vertex on the shortest path:
```

Comment:
When you execute this query, you receive a result as an array where each element represents a vertex in the shortest path and consists of two items:

Reply:
@martin-macrometa, I made the change, but I'm not sure it makes sense. How can "each element represents a vertex in the shortest path" and the results being an edge and the vertex both be true?